### PR TITLE
Add grouping for checks

### PR DESCRIPTION
**Pull request by captnCC**
This allows checks to be assigned to groups and to run checks of a single group via the `RunHealthChecksCommand`.

This should be only breaking if the `shouldRun` method of a check has been overridden in a custom check.

### CODE REVIEW

File: src/Commands/RunHealthChecksCommand.php [hunk 1]
```diff
@@ -26,7 +26,13 @@ class RunHealthChecksCommand extends Command
 
     public function handle(): int
     {
-        $this->info('Running checks...');
+        $group = $this->option('group');
+
+        if (is_string($group)) {
+            $this->info("Running checks for group ${group}...");
+        } else {
```
[the remark this answers is not in the document]
Comment (captnCC) [reply src/Commands/RunHealthChecksCommand.php]: An early return would be beneficial at this point, but I merged the two `$this->info()` calls with an shorthand if to clean things a little up

**Comment by captnCC** [comment]
The test are now passing.
On the way I found another bug regarding skipped jobs: The `ended_at` attribute was missing from the `Result` instance and caused the result storing to fail. To fix it I added the `ended_at` timestamp when the skipped result gets created.

**Comment by freekmurze** [comment]
Thanks for your work on this, however it seems that the tests are still failed...

**Comment by captnCC** [comment]
Weirdly enough, the failed test isn't even related to my changes and only failed in the P8.1/L8 configuration for the PR. In my fork the test worked fine. Maybe it's just a fluke in that run. 
Has GitHub the option to rerun actions?

**Comment by freekmurze** [comment]
If you make a small change somewhere, I think I'll be able to run the tests again.

**Comment by freekmurze** [comment]
In the interest in keeping this package as simple as it can be, and because not too many people are asking for this feature, I've decided to pass on it for now.

**Pull request closed without merging**